src/controllers/request.controller.js

```diff
@@ -0,0 +1,25 @@
+import RequestService from '../services/request.service';
```

Comment:
I see there's a lot of similarities between this PR and #42. See the PR for the feedback I left there and implement them here.

Reply:
Yeah, they are almost similar. ok let me do it.